node/subsystem-util/src/lib.rs

```diff
@@ -442,9 +442,12 @@ impl<ToJob: ToJobTrait> JobHandle<ToJob> {
 
 /// This module reexports Prometheus types and defines the [`Metrics`] trait.
 pub mod metrics {
-	/// Reexport Prometheus types.
+	/// Reexport prometheus crate
+	pub use ::prometheus as prometheus_super;
```

Comment:
IIUC, this is needed because substrate_prometheus_endpoint does not expose `HistogramTimer`
https://github.com/paritytech/substrate/blob/9ec43d880823e9ad082de7e3038ac9777613fa15/utils/prometheus/src/lib.rs#L18-L27

Do you think we should fix that?

Reply:
You understand correctly. My motivation for making the change here instead of in substrate was simply that it's a lot less work to encapsulate all the changes within a single repo, instead of synchronizing cross-repo PRs.

Reply:
Actually, I think it does via `self`, we could write `use metrics::prometheus::prometheus::HistogramTimer`, which is cumbersome, but works.

Reply:
Good catch! 2b752e4e.